packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts

```diff
@@ -1,12 +1,13 @@
 import {
+  PLUGIN_MANIFEST,
   REMOTE_ENTRY_SCRIPT,
   REMOTE_ENTRY_CALLBACK,
-  extensionArraySchema,
-} from '@openshift/dynamic-plugin-sdk';
-import type { Extension } from '@openshift/dynamic-plugin-sdk';
+} from '@openshift/dynamic-plugin-sdk/src/constants';
+import { extensionArraySchema } from '@openshift/dynamic-plugin-sdk/src/yup-schemas';
+import type { EncodedExtension } from '@openshift/dynamic-plugin-sdk/src/types/extension';
 import * as _ from 'lodash-es';
```

Comment:
I would use `lodash` here directly since we are exposing `cjs` package. And while on it we can import just what we use in here.

Reply:
@karelhala For consistency, I'd prefer to use `lodash-es` everywhere in our sources, and adapt generated outputs as necessary.

With `lodash-es` you can do `import * as _` and the module bundler like Rollup will still be able to figure out which parts of this library are actively used in the compilation. Agreed that it'd be good to look into a good convention on how to handle Lodash imports, but I'd do it as a follow-up :small_airplane: